modules/placement-driver-api/src/main/java/org/apache/ignite/internal/placementdriver/ReplicaMeta.java

```diff
@@ -19,29 +19,19 @@
 
 import java.io.Serializable;
 import org.apache.ignite.internal.hlc.HybridTimestamp;
+import org.jetbrains.annotations.Nullable;
 
-/**
- * Replica lease meta.
- */
+/** Replica lease meta. */
 public interface ReplicaMeta extends Serializable {
-    /**
-     * Gets a leaseholder node consistent ID.
-     *
-     * @return Leaseholder or {@code null} if nothing holds the lease.
-     */
-    String getLeaseholder();
+    /** Gets a leaseholder node consistent ID (assigned to a node once), {@code null} if nothing holds the lease. */
+    @Nullable String getLeaseholder();
+
+    /** Gets a leaseholder node ID (changes on every node startup), {@code null} if nothing holds the lease. */
+    @Nullable String getLeaseholderId();
```

Comment:
Where is the property used in the production code?

Reply:
I need to restore the index build so that there is no bug described in the ticket description.
Please see javadoc in https://github.com/apache/ignite-3/pull/2741/commits/9e1eb26f8c8b85be8c0f97e7142c2af7ec9f4fc4